templates/default/jaas-activedirectory.conf.erb

```diff
@@ -29,7 +29,11 @@ activedirectory {
 <% end -%>
     reportStatistics="<%=@ldap[:reportstatistics]%>";
 
-    org.eclipse.jetty.plus.jaas.spi.PropertyFileLoginModule required
+    <% if Gem::Requirement.new('< 2.7.0').satisfied_by?(Gem::Version.new(@rundeck_version)) %>
```

Comment:
I dont think `@rundeck_version` is available in this template - https://github.com/Webtrends/rundeck/blob/master/recipes/server_install.rb#L155-L160

can you spec this out and prove that the file contains `org.eclipse.jetty.plus.jaas.spi.PropertyFileLoginModule sufficient` or `org.eclipse.jetty.jaas.spi.PropertyFileLoginModule sufficient` when its supposed to? That should be easy with ChefSpec:

```ruby
expect(chef_run).to render_file('/etc/rundeck/jaas-activedirectory.conf').with_content('the content')
```